techniques/failures/F85.html

```diff
@@ -23,14 +24,14 @@ <h3>Setting focus to the document after dismissing a menu embedded on the page</
                <p>Activate the trigger control via the keyboard.</p>
                <ul>
                   <li>Check whether focus is in the menu or dialog.</li>
-                  <li>Check whether advancing the focus in the sequential navigation order puts focus in the menu or dialog.</li>
+                  <li>Check whether moving the focus forwards once in the sequential navigation order puts focus in the menu or dialog.</li>
                </ul>
             </li>
             <li>
-               <p>Dismiss the menu or dialog.</p>
+               <p>Dismiss the menu or dialog</p>
                <ul>
                   <li>Check whether focus is on the trigger control.</li>
-                  <li>Check whether advancing the focus backwards in the sequential navigation order puts focus in the trigger control.</li>
+                  <li>Check whether moving the focus backwards once in the sequential navigation order puts focus in the trigger control.</li>
```

Comment:
"focus in" > "focus on" ?

Reply:
Can't this be said in less technical language?